docs/migration_command_pattern.rst

```diff
@@ -92,26 +77,19 @@ did it run successfully?) can be performed in the context of a Django migration.
             return
 
         if num_items < AUTO_MIGRATE_ITEMS_LIMIT:
-            try:
-                call_command(
-                    "the_migration_management_command",
-                    dbname=schema_editor.connection.alias,
-                )
-                migrated = count_items_to_be_migrated(schema_editor.connection) == 0
-                if not migrated:
-                    print("Automatic migration failed")
-            except Exception:
-                traceback.print_exc()
+            safely_run_management_command(
+                "the_migration_management_command",
+                dbname=schema_editor.connection.alias,
+                required_commit=GIT_COMMIT_WITH_MANAGEMENT_COMMAND
+            )
+            remaining = count_items_to_be_migrated(schema_editor.connection)
+            if remaining != 0:
+                print(f"Automatic migration failed, {remaining} items remain to migrate.")
+                sys.exit(1)
         else:
             print("Found %s items that need to be migrated." % num_items)
             print("Too many to migrate automatically.")
```

Comment:
This example is broken. This branch used to ultimately lead to `sys.exit(1)`.

Overall, I don't like the way this has changed. The instructions are less helpful if you hit the `Automatic migration failed, {remaining} items remain to migrate` branch above. The code does not do what it used to do.